src/lug/Graphics/Vulkan/Render/Window.cpp

```diff
@@ -41,13 +42,16 @@ bool Window::pollEvent(lug::Window::Event& event) {
             }
         }
 
+        _guiInstance->processEvents(event);
+
         return true;
     }
 
     return false;
 }
 
-bool Window::beginFrame() {
+bool Window::beginFrame(const lug::System::Time &elapsedTime) {
+    _guiInstance->beginFrame(elapsedTime);
```

Comment:
It's probably better to do that at the end of the function (during the function we kind of refresh the swapchain if it is outdated), and you also need to check the return of the function because it returns a `bool`.